posthog/admin/inlines/totp_device_inline.py

```diff
@@ -5,3 +5,5 @@
 class TOTPDeviceInline(admin.TabularInline):
     model = TOTPDevice
     extra = 0
+    # only include non-sensitive fields
+    fields = ("name", "confirmed", "throttling_failure_timestamp", "throttling_failure_count", "last_used_at")
```

Comment:
Nitpick: would it add value to make them read-only, or are they read-only by default? 👀

Reply:
They are not yet read-only. I was wondering about this when auditing our Admin models - should _most_ fields be read-only? I don't yet have a great understanding of the primary support use case for these pages.